src/lib/crypto/signatures.cpp

```diff
@@ -175,6 +193,21 @@ signature_calculate(pgp_signature_t &     sig,
         }
         break;
     }
+#if defined(ENABLE_PQC)
+    case PGP_PKA_DILITHIUM3_ED25519:
+        [[fallthrough]];
+    // case PGP_PKA_DILITHIUM5_ED448: [[fallthrough]];
```

Comment:
Generally we fully agree with not having commented-out code in the repository. This is a bit of a special case, as Ed448 should be added as soon as it becomes available in Botan and having the case-statements already as comments will make it easier to add it later. 

One possiblity I could imagine would be to comment in the code and throw an error for now in case Curve448 is encountered. Would that make more sense to you?

Reply:
Throwing an error in case which is not supposed to work at all doesn't seem better :) Another possible solution is to change it to `// TODO: add case PGP_PKA_DILITHIUM5_ED448: [[fallthrough]];`, so it doesn't look like commented code but as todo item.